frontend/integration-tests/views/kubernetes-marketplace.view.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-undef, no-unused-vars */
+
+import { browser, $, $$, ExpectedConditions as until } from 'protractor';
+
+export const entryTiles = $$('.catalog-tile-pf');
+export const entryTileFor = (name: string) => entryTiles.filter((tile) => tile.$('.catalog-tile-pf-title').getText()
+  .then(text => text === name)).first();
+export const entryTileCount = (name: string) => entryTiles.filter((tile) => tile.$('.catalog-tile-pf-title').getText()
+  .then(text => text === name)).count();
+
+// FilterSidePanel views
+export const filterCheckboxes = $$('.filter-panel-pf-category-item');
+// Using string  includes() since filter provider names aren't standard
+export const filterCheckboxFor = (name: string) => filterCheckboxes.filter((filterItem) => filterItem.$('label').getText()
+  .then(text => text.toLowerCase().includes(name.toLowerCase()))).first();
+export const clickFilterCheckbox = (name: string) => filterCheckboxFor(name).$('input').click();
+export const activeFilterCheckboxes = filterCheckboxes.filter((filterItem) => filterItem.$('input').isSelected()
+  .then(bool => bool));
+export const filterTextbox = $$('.filter-panel-pf-category').first().$('input')
+export const filterByName = (filter: string) => filterTextbox.sendKeys(filter);
+export const clearFiltersText = $('.co-catalog-page__no-filter-results').$('.blank-slate-pf-helpLink').$('button');
+
```

Comment:
Can we move everything above to a new, shared `catalog-page.view.ts`? `kubernetes-marketplace.view.ts` can keep MarketPlaceUI specific objects (everything below) and `import catalog-page.view.ts`. 
In my PR I renamed `catalog.view.ts` to a more specific `olm-catalog.view.ts`. The new `catalog.view.ts` is for Home -> Catalog, can import the new, shared `catalog-page.view.ts` (so MarketPlace and Home-Catalog use the same catalog page tiles and filter objs) and contain specific objects for Home -> Catalog. make sense? -thanks

Reply:
Or should I create the new `catalog-page.view.ts`?  I guess it depends on who we think will merge first.

Reply:
That makes sense, since the above objects are used for CatalogTileView and FilterSidePanel components.